src/stories/Library/event-list-item/EventListItem.tsx

```diff
@@ -0,0 +1,49 @@
+// EventListItem.tsx
+import { ReactComponent as ArrowSmallRight } from "../Arrows/icon-arrow-ui/icon-arrow-ui-small-right.svg";
+import { Tag } from "../tag/Tag";
+
+export type EventListItemProps = {
+  image: string;
+  title: string;
+  description: string;
+  date: string;
+  time: string;
+  location: string;
+  price: string;
+  href: string;
+};
+
+export const EventListItem: React.FC<EventListItemProps> = ({
+  image,
+  title,
+  description,
+  date,
+  time,
+  location,
+  price,
+  href,
+}) => {
+  return (
+    <a href={href} className="event-list-item arrow__hover--right-small">
+      <div className="event-list-item__image-container">
+        <img src={image} alt={title} className="event-list-item__image" />
+      </div>
+      <div className="event-list-item__content">
+        <Tag hasBackground className="event-list-item__tag">
+          foredrag
+        </Tag>
```

Comment:
Please make the tag configurable as well.

I cannot tell why it should not be.